devices/Ft6xx6x/samples/Program.cs

```diff
@@ -1,55 +1,226 @@
-// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Need to keep the license header here

Reply:
ok, sorry for that , i dont know why i did not copied it too..